docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3"
+services:
+  posix:
+    build: 
+      context: .
+      dockerfile: ./Dockerfile.dev
+    volumes:
+      - ./:/app
+    ports:
+      - "${POSIX_PORT}:${POSIX_PORT}"
+    command: ["sh", "-c", CompileDaemon -build="go build -C ./cmd/versitygw -o versitygw" -command="./cmd/versitygw/versitygw -p :$POSIX_PORT -a $ACCESS_KEY_ID -s $SECRET_ACCESS_KEY --iam-dir /tmp/vgw posix /tmp/vgw"]
+  proxy:
+    build: 
+      context: .
+      dockerfile: ./Dockerfile.dev
+    volumes:
+      - ./:/app
+    ports:
+      - "${PROXY_PORT}:${PROXY_PORT}"
+    command: ["sh", "-c", CompileDaemon -build="go build -C ./cmd/versitygw -o versitygw" -command="./cmd/versitygw/versitygw -a $ACCESS_KEY_ID -s $SECRET_ACCESS_KEY -p :$PROXY_PORT s3 --endpoint http://posix:$POSIX_PORT"]
```

Comment:
Tangential question as I don't see it in the docs. Whats the difference between proxy and posix? Can you scale both up independently?

Reply:
s3proxy is missing in the docs right now. We were working on finalizing functionality before documenting and having people use it.  The backend storage for the gateway is an s3 server for the s3proxym (vs a filesystem for posix).  The original use case was for us to be able to compare performance of direct to s3 storage vs going through the gateway. But there has been interest in using the s3 backend to manage local IAM accounts and have a single backend storage account. This way a site admin could have a single account on an s3 service, and then manage access with local IAM accounts.

Reply:
Ah. Gotcha. Thanks.